### ISSUE TITLE
Are we allowed to fix bugs discovered by our team?

### ISSUE THREAD

**Issue by winstonlimjy** [nus-cs2103-AY2425S1/forum#636]
Hi, are we able to fix the bugs discovered by our team instead of other teams from the PE-D? `Bugs regarding input validations`

**Comment by damithc** [comment]
@winstonlimjy Yes, you can. Who discovered the bug doesn't matter but you still need to adhere to the feature freeze restrictions.
